nengo_dl/tests/test_nengo_tests.py

```diff
@@ -12,16 +12,6 @@
 from nengo_dl.tests import dummies
 
 
-def test_warn_on_opensim_del(Simulator):
-    with nengo.Network() as net:
-        nengo.Ensemble(10, 1)
-
-    sim = Simulator(net)
-    with pytest.warns(RuntimeWarning):
-        sim.__del__()
-    sim.close()
```

Comment:
It's a copy of a test in NengoCore that had to be updated for newer pylint, figured there's no point in updating when we could remove instead.

Reply:
It's slightly different than the NengoCore test, because the NengoCore simulator raises a `ResourceWarning` (which is hidden by default) whereas NengoDL raises a `RuntimeWarning` (visible by default, since it's more important in NengoDL that people aren't leaving around unclosed simulators).

Reply:
Ah I see. Yeah should bring it back then.